app/assets/javascripts/routers/management/PageEditionRouter.js

```diff
@@ -4,9 +4,10 @@
   /* eslint-disable no-unused-vars */
   var OPEN_CONTENT = 1;
   var ANALYSIS_DASHBOARD = 2;
-  var DYNAMIC_INDICATOR = 3;
-  var LINK = 6;
-  var STATIC_CONTENT = 7;
+  var HOMEPAGE = 3;
```

Comment:
Indeed! I didn't want to change it without asking @clementprdhomme. Clément, do you mind checking this? :)